apps/src/lab2/types.ts

```diff
@@ -336,9 +336,11 @@ export interface Condition {
   value?: string | number;
 }
 
+type ValueType = 'string' | 'number';
+type ConditionValueType = `${ValueType}:${ValueType}` | ValueType;
```

Comment:
🪄

Reply:
Haha, I kind of couldn't believe it worked!